src/pages/Overview/AnalyticsBanner.tsx

```diff
@@ -1,23 +0,0 @@
-import { FC } from "react"
-import { useColorModeValue } from "@threshold-network/components"
-import analyticsImageLight from "../../static/images/AnalyticsIllustration.png"
-import analyticsImageDark from "../../static/images/AnalyticsIllustrationDark.png"
-import { ModalType } from "../../enums"
-import { useModal } from "../../hooks/useModal"
-import SecondaryAnnouncementBanner from "../../components/AnnouncementBanner/SecondaryAnnouncementBanner"
-
-export const AnalyticsBanner: FC = () => {
```

Comment:
Since we are removing `AnalyticsBanner` can we also remove `Feedback` page in this PR? I see that we don't use it on production anyway (FEEDBACK MODULE feature flag is set to false).

Reply:
Nevermind. I see you addressed it in a [separate PR](https://github.com/threshold-network/bitcoin-on-base/pull/11). Let's keep it that way then.